### DISCUSSION TITLE
Update Failed

### DISCUSSION THREAD

**Discussion by PeterKnotek** [mattieserver/netbox-topology-views#443] [Q&A] [answered]
Hi

After update Netbox to latest version, have problem update netbox topology.
Still install version (3.8.1) not version 3.9.0 which is required for netbox VER 3.7.0

Does anyone have the same problem??

**Comment by mattieserver** [comment, marked as answer]
Hi,

Please use `netbox-topology-views==3.9b1`.

Support for 3.7.0 is still in 'beta'.
